src/ex4/dist/clients/itemClient.js

```diff
@@ -0,0 +1,67 @@
+// Create an ItemClient class here. This is what makes requests to your express server (your own custom API!)
```

Comment:
make sure to keep the file names in one convention 
you have sometimes error_handle and sometimes errorHandler
the convention is error_handle